frontend/pages/index.tsx

```diff
@@ -1,7 +1,19 @@
-import { Heading, Link } from "@chakra-ui/react";
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/unbound-method */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
We don't need these anymore! There was a fix for these typescript errors previously, so we can just remove it from here